api/cerbos/svc/v1/svc.proto

```diff
@@ -63,6 +63,16 @@ service CerbosService {
             description: "Check a principal's permissions to a batch of heterogeneous resources and actions."
         };
     }
+
+    rpc ServerInfo(cerbos.request.v1.ServerInfoRequest) returns (cerbos.response.v1.ServerInfoResponse) {
+        option (google.api.http) = {
+            get: "/api/status"
```

Comment:
This should be `/api/server_info` because `status` means something else in this context.

Reply:
Right, I was using the same command as before which included the username and password 🤦